scripts/test-run.sh

```diff
@@ -28,21 +28,11 @@ TEST_ENV_NAME="${TEST_ENV_NAME:-kong-charts-tests}"
 KUBECTL="kubectl --context kind-${TEST_ENV_NAME}"
 KUBERNETES_VERSION="$($KUBECTL version -o json | jq -r '.serverVersion.gitVersion')"
 
-CONTROLLER_PREFIX=""
-GATEWAY_PREFIX=""
 ADDITIONAL_FLAGS=()
 
 # ------------------------------------------------------------------------------
-# Configure per-chart settings
+# Deploy Kuma configuration and test namespace
```

Comment:
Is there anything to do with kuma?

Reply:
Nope, there's another block lower that does that has the same comment, this was some garbage I didn't catch when reviewing the conflicts. Good catch.